rotkehlchen/chain/ethereum/adex/adex.py

```diff
@@ -461,20 +497,114 @@ def _deserialize_channel_withdraw(
         )
 
     @staticmethod
-    def _deserialize_adex_event(
+    def _deserialize_adex_channel_event(
+            raw_event: Dict[str, Any],
+            inverse_identity_address_map: Dict[ChecksumAddress, ChecksumAddress],
+            case: Literal['channel_withdraw'],
```

Comment:
What is this function for? This is just used for channel withdraw so why pass the literal case here with only 1 value? Also what I wrote above about not deserializing the log index again both in here and outside becomes even more puzzling. If there is more events we need to parse in the future it makes sense. If not then it does not.